### PR TITLE
add rec ctc model for tipc

### PR DESCRIPTION
**Pull request by tink2123**
att

### CODE REVIEW

File: test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/rec_icdar15_train.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+PostProcess:
+  name: CTCLabelDecode
+
+Metric:
+  name: RecMetric
+  main_indicator: acc
+
+Train:
+  dataset:
+    name: LMDBDataSet
```
Comment (LDOUBLEV) [inline test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/rec_icdar15_train.yml:61]: 应该是simpledataset

Comment (tink2123) [reply test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/rec_icdar15_train.yml]: done, thx

File: test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/train_infer_python.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
+===========================train_params===========================
+model_name:ocr_rec
+python:python3.7
+gpu_list:0|0,1
+Global.use_gpu:True|True
+Global.auto_cast:null
+Global.epoch_num:lite_train_infer=2|whole_train_infer=300
```
Comment (LDOUBLEV) [inline test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/train_infer_python.txt:7]: Mode需要修改

lite_train_lite_infer
whole_train_whole_infer

Comment (tink2123) [reply test_tipc/configs/rec_mv3_none_bilstm_ctc_v2.0/train_infer_python.txt]: done, thx

**Review by LDOUBLEV — APPROVED** [review]

**Review by LDOUBLEV — APPROVED** [review]

**Pull request merged**